dali/pipeline/operators/resize/resize_test.cc

```diff
@@ -16,35 +16,105 @@
 
 namespace dali {
 
+// Values of eps for different tests
+//      first column:   average deviation is calculated combined for all images
+//      second column:  average deviation is calculated separately for each image
+static double testEps[] = {
+                    0.7, 1.8,
```

Comment:
Please add a comment what eps is matches with what test.

Reply:
DONE. New 24 tests were added with the calculation of average deviation separately for each image of the batch.